products/citrix-apps-desktops.md

```diff
@@ -1,98 +1,117 @@
 ---
 title: Citrix Virtual Apps and Desktops
+category: app
+iconSlug: citrix
 permalink: /citrix-vad
-changelogTemplate: https://docs.citrix.com/en-us/citrix-virtual-apps-desktops/__RELEASE_CYCLE__/whats-new.html
 alternate_urls:
 -   /citrixvad
 -   /cvad
   # This can be removed if we add more citrix products
 -   /citrix
-iconSlug: citrix
-category: app
 releasePolicyLink: https://www.citrix.com/support/product-lifecycle/product-matrix.html
+changelogTemplate: https://docs.citrix.com/en-us/citrix-virtual-apps-desktops/__RELEASE_CYCLE__/whats-new.html
+
+LTSLabel: "<abbr title='Long Term Service Release'>LTSR</abbr>"
 activeSupportColumn: true
 releaseColumn: true
 releaseDateColumn: true
-LTSLabel: "<abbr title='Long Term Service Release'>LTSR</abbr>"
+extendedSupportColumn: true
+
 releases:
 -   releaseCycle: "2212"
-    latest: "2212"
+    releaseDate: 2022-12-19
     support: 2023-06-19
     eol: 2024-06-19
```

Comment:
When `extendedSupportColumn` is active, the `extendedSupport` must be specified for all release cycles (same constraint as `activeSupportColumn`).

Reply:
Fixed.